conf/scripts/generate-saml-cert.sh

```
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -euo pipefail
```

Shellcheck (0.10.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)